### ISSUE TITLE
Python 3.10 Support

### ISSUE THREAD

**Issue by damccorm** [apache/beam#21585]
Support for Python 3.10

Imported from Jira [BEAM-14126](https://issues.apache.org/jira/browse/BEAM-14126). Original Jira may contain additional context.
Reported by: Anand Inguva.

**Comment by nharada1** [comment]
Is Beam going to get Python 3.10 support in the near term? More broadly, how soon does Beam aim to support new Python releases?

**Comment by kennknowles** [comment]
@AnandInguva if I recall, you were considering working on this
/cc @tvalentyn

**Comment by AnandInguva** [comment]
Yes, its in progress. It's blocked on few type hint related things.

**Comment by AnandInguva** [comment]
@tvalentyn can add more pointers on the release dates.

**Comment by AnandInguva** [comment]
@tvalentyn @damccorm Would you be able to close this? This is a duplicate of https://github.com/apache/beam/issues/21458

**Comment by damccorm** [comment]
Sure - you can also close it with chat commands:

.close-issue

**Comment by AnandInguva** [comment]
Closing this as this is a duplicate of #21458 

.close-issue

**Comment by AnandInguva** [comment]
.close-issue

**Comment by damccorm** [comment]
(GitHub Actions are a little backed up right now, so it might take a little while - see the queue here https://github.com/apache/beam/actions)
